project/LinkerdBuild.scala

```diff
@@ -83,6 +83,16 @@ object LinkerdBuild extends Base {
       .dependsOn(core)
       .withTests()
 
+
+    object Identifier {
+      val http = projectDir("linkerd/identifier/http")
+        .dependsOn(core, Router.http)
+        .withTests()
```

Comment:
What about additional identifiers that aren't the default? I was thinking of writing some identifiers to replace rewriting namers. Would they also go in protocol.http or should they stay here?